src/notification/notification.component.ts

```diff
@@ -86,10 +87,15 @@ export class Notification {
 	@HostBinding("class.bx--inline-notification--success") get isSuccess() { return this.notificationObj.type === "success"; }
 	@HostBinding("class.bx--inline-notification--warning") get isWarning() { return this.notificationObj.type === "warning"; }
 
+	get isShowClose() {
```

Comment:
We could probably just leave this as `showClose` ... or rename it to something that makes sense grammatically 👍